blockchain/My-Wallet-V3-iOS
# feat(Onfido): IOS-1162 create onfido applicant and retrieve credentials

## Objective

Creating Onfido applicant and retrieving credentials from Nabu so that we can start the Onfido flow.

## Description

Worked off where @achtenhagen left off to create an Onfido applicant by interacting directly with the [Onfido API](https://documentation.onfido.com/#create-applicant), followed by requesting an Onfido credential from Nabu. This unblocks our Onfido integration so now you can go through that flow (i.e. capture an ID w/ the camera and a selfie).

...looks like our Onfido integration isn't quite complete yet though but will get to that next.

## How to Test

Pull in these changes, go through the KYC flow, and once you get to the "Verify Your Identity" screen, you should be able to pick an ID type followed by going through the Onfido flow.

## Merge Checklist
